tools/jbrowse2/jbrowse2.py

```diff
@@ -1146,10 +1171,10 @@ def add_paf(self, data, trackData, pafOpts, **kwargs):
             if gname not in self.genome_names:
                 # ignore if already there - eg for duplicates among pafs.
                 asstrack, first_contig = self.make_assembly(gpath, gname, useuri)
-                self.genome_names.append(gname)
+                self.genome_names.append(copy.copy(gname))
```

Comment:
why is the hard-copy needed?

Reply:
in that string case it's not - that's a wart I'll fix thanks - I got overenthusiastic making copies of mutables to  appended to generate json. Was seeing strange interactions I think.